.travis.yml

```diff
@@ -6,7 +6,9 @@ python:
 install:
   - pip install -r requirements.txt
   - pip install typelint
+  - pip install black
 # command to run tests
 script:
   - python code/ingest.py || python3 code/ingest.py
   - python -m typelint --quiet code/ingest.py
+  - if !((python -m black --check)) ; then echo "Black formatter is required!"; fi
```

Comment:
you don't need the `if` or the `echo`, you can literally run `python -m black --check` and then the Travis CI build will fail when this fails.

Reply:
Done!